packages/swapper/src/swappers/osmosis/OsmosisSwapper.ts

```diff
@@ -357,7 +365,7 @@ export class OsmosisSwapper implements Swapper<ChainId> {
     const cosmosAddress = isFromOsmo ? receiveAddress : sellAddress
     const signTxInput = await buildTradeTx({
       osmoAddress,
-      accountNumber,
+      accountNumber: receiveAccountNumber,
```

Comment:
LGTM, this was super confusing to read haha:

https://github.com/shapeshift/lib/blob/d76db7a521277d7d2cb6950cfe86494edc7948f1/packages/swapper/src/swappers/osmosis/utils/helpers.ts#L297-L298